pkg/testutil/nerdtest/utilities.go

```diff
@@ -29,6 +29,14 @@ import (
 	"github.com/containerd/nerdctl/v2/pkg/testutil/test"
 )
 
+const (
+	// It seems that at this moment, the busybox on windows image we are using has an outdated version of sleep
+	// that does not support inf/infinity.
+	// This constant is provided as a mean for tests to express the intention of sleep infinity without having to
+	// worry about that and get windows compatibility.
+	Infinity = "3600"
```

Comment:
This is a finite number 😄

Reply:
It's an infinitely finite number, right? :P

Reply:
Maybe just use "18446744073709551615" (or "4294967295" if it overflows)?
It should be also moved to `*_windows.go`

Reply:
I wonder if this is worth the effort.

For all intents and purposes, these containers never live more than a few seconds.
Furthermore, having two distinct behaviors for linux (`infinity`) and windows (`4294967295`) is inviting trouble.
Finally, this is just a workaround for a situation that hopefully will resolve eventually.

wdyt?